nyc_data/xlsx_utils.py

```diff
@@ -54,12 +52,16 @@ def guess_mapping(sheet: Path, possible_mappings: List[SheetMapping]):
         if mapping.sheet_name is not None and workbook:
             sheet = workbook[mapping.sheet_name]
             first_row = next(XLSXDictReader(sheet))
+
         else:
 
-            with open(sheet, "r", encoding="utf16") as csvfile:
-                text = csvfile.read()
-            reader = csv.DictReader(text.splitlines())
-            first_row = next(reader)
+            try:
+                with open(sheet, encoding="latin-1") as csvfile:
+                    reader = csv.DictReader(csvfile)
+                    first_row = next(reader)
+                    
+            except Exception as exc:
+                raise data_import.CsvImportError('Error reading in CSV file')
```

Comment:
this is a circular import, and we probably don't want to completely swallow the original exception. You can do `raise CsvImportError(...) from exc` to set a cause and preserve the underlying exception. Anyway, you can move all the exceptions into `mapper/types.py` to break the circular dependency.

Reply:
OK, I'll chew on that a bit and get to it asap (probably tomorrow)